addons/survey/models/survey_survey.py

```diff
@@ -770,8 +769,7 @@ def _get_next_page_or_question(self, user_input, page_or_question_id, go_back=Fa
                     if contains_active_question or is_description_section:
                         return question
                 else:
-                    triggering_answers = triggering_answers_by_question.get(question)
-                    if not triggering_answers or triggering_answers & selected_answers:
+                    if question not in inactive_questions:
```

Comment:
Hello @LightSystem 
Thanks for the PR!
It's absolutely correct, `_get_inactive_conditional_questions` is already calling the above methods and doing this exact check.

We'd consider it an [REF] commit, indeed targeting `master` as it's not correcting a bug, and we want to keep supporting customizations (that could theoretically have implemented a different logic using these methods so we shouldn't remove the "hooks").

Reply:
Thanks for the review! I'll change the commit to `[REF]` and rebase on `master`.